cmds/bmp/bms_command.go

```diff
@@ -1,9 +1,19 @@
 package bmp
 
 import (
+	"errors"
+	"gopkg.in/yaml.v2"
+	"path/filepath"
+	"io/ioutil"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
 	clients "github.com/cloudfoundry-community/bosh-softlayer-tools/clients"
 	cmds "github.com/cloudfoundry-community/bosh-softlayer-tools/cmds"
 	common "github.com/cloudfoundry-community/bosh-softlayer-tools/common"
+	tablewriter "github.com/olekukonko/tablewriter"
```

Comment:
Hi Dmitriy @cppforlife，
Max mentioned you have a table writer, too. Could you please share the link with me and I would like to have a try:)
Thanks in advance!